mobile_app/README.md

```
@@ -0,0 +1,53 @@
+
+Мобильное приложение Garincha Group реализовано на **Expo/React Native** и покрывает ключевые сценарии, описанные в корневом `README.md` репозитория.
+
+## Возможности
+- Каталог предложений партнёров холдинга и карточка товара
+- Управление корзиной и оформление заказа
+- Личный кабинет участника клуба с настройкой профиля и уведомлений
+- Раздел бонусной программы с историей операций и прогнозом начислений
+- Авторизация участника и хранение данных в состоянии приложения
+
+## Структура
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)